src/types/operation.ts

```diff
@@ -7,28 +7,33 @@ export type OperationType =
   | "NONE"
   | "CREATE"
   | "REVEAL"
+  // COSMOS
   | "DELEGATE"
   | "UNDELEGATE"
   | "REDELEGATE"
   | "REWARD"
-  | "FEES" // TRON
+  // TRON
+  | "FEES"
   | "FREEZE"
   | "UNFREEZE"
-  | "VOTE" // POLKADOT
+  // POLKADOT
+  | "VOTE"
   | "REWARD_PAYOUT"
   | "BOND"
   | "UNBOND"
   | "WITHDRAW_UNBONDED"
   | "SET_CONTROLLER"
   | "SLASH"
   | "NOMINATE"
-  | "CHILL" // COMPOUND TYPE OPERATIONS
+  // COMPOUND TYPE OPERATIONS
+  | "CHILL"
   | "SUPPLY"
   | "REDEEM"
   | "APPROVE"
+  // ALGORAND
   | "OPT_IN"
   | "OPT_OUT"
-  | "CLOSE_ACCOUNT"
```

Comment:
was this not used?

Reply:
Nope indeed, was never implemented neither in CLI or LLD & LLM. Just did some cleanup.